specifications/providers/openstack/flavors_and_images.md

```diff
@@ -126,6 +126,10 @@ As defined by this specification we expect:
 ```json
 {
   "properties": {
+    "unikorn:os_name": "Ubuntu",
+    "unikorn:os_version": "24.04",
+    "unikorn:os_family": "Debian",
+    "unikorn:tags": "slurmd=24.11,wireguard=1.0",
```

Comment:
This kinda worries me that it can be overloaded and we'd have trouble telling packages from actual metadata.  As such, I reckon we go with `packages` for now?